index.html

```diff
@@ -56,25 +62,216 @@ <h1>
+              <li>
+                <a href="#"
+                  ><img src="./images/medium.svg" alt="medium-logo"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="./images/github.svg" alt="github-logo"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="./images/angelist.svg" alt="angelist-logo"
+                /></a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </section>
+
+      <section id="works">
+        <div class="card">
+          <div class="head-image">
+            <img
+              src="./images/Snapshoot Portfolio.png"
+              alt="work-image-project"
+            />
+            <h1 class="tonic">TONIC</h1>
+            <ul class="dates">
+              <li>CANOPI</li>
+              <li><img src="./images/Counter (1).svg" alt="counter" /></li>
+              <li class="stack-backend">Back End Dev</li>
+              <li><img src="./images/Counter (1).svg" alt="counter" /></li>
+              <li class="stack-backend">2015</li>
+            </ul>
+
+            <p class="paragraph">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="stacks">
+              <li class="stack-over">HTML</li>
+              <li class="stack-over">CSS</li>
+              <li class="stack-over">Javascript</li>
             </ul>
+            <a href="#" class="btn">See Project</a>
           </div>
         </div>
-          
-       
+        <div class="card">
+          <div class="head-image">
+            <img src="./images/Snapshoot 2.png" alt="work-image-project" />
+            <h1 class="tonic">Multi-Post <br />Stories</h1>
+            <ul class="dates">
+              <li>CANOPI</li>
+              <li><img src="./images/Counter (1).svg" alt="counter" /></li>
+              <li class="stack-backend">Back End Dev</li>
+              <li><img src="./images/Counter (1).svg" alt="counter" /></li>
+              <li class="stack-backend">2015</li>
+            </ul>
+
+            <p class="paragraph">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="stacks">
+              <li class="stack-over">HTML</li>
+              <li class="stack-over">CSS</li>
+              <li class="stack-over">Javascript</li>
+            </ul>
+            <a href="#" class="btn">See Project</a>
+          </div>
+        </div>
+        <div class="card">
+          <div class="head-image">
+            <img src="./images/Snapshoot 3.png" alt="work-image-project" />
+            <h1 class="tonic">TONIC</h1>
+            <ul class="dates">
+              <li>CANOPI</li>
+              <li><img src="./images/Counter (1).svg" alt="counter" /></li>
```

Comment:
- [ ] Kindly work on the images you name ./mages/Counter (1),svg kindly fix this not display from my ends you can recheck it pls and fix it. 

- [ ] your design 

 
![broken design](https://user-images.githubusercontent.com/92458236/235310392-5fae22e4-58df-4ea2-ba75-c2e7e420c65e.jpg)